client/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -81,10 +83,61 @@ describe('FeedbackGeneratorService', function() {
             c: 'j'
           })
         ).toEqual('{"a": 3, "b": 5, "c": "j"}');
+        // Need to define and call blankFunction for 100% karma coverage
```

Comment:
I am trying to cover the case when jsToHumanReadable goes to the else branch, so I am calling a function. However, if I simply pass in a blank function, the lint script complains, and if I create  a function that does nothing, I need to call it explicitly otherwise the line inside the function doesn't get code coverage. Not sure if this is the best workaround